### ISSUE TITLE
bug(linter): no-undef false positive

### ISSUE THREAD

**Issue by u9g** [oxc-project/oxc#736]
https://gist.github.com/u9g/d91dc13637d2af1cf5eef2ddc7a0814b

**Comment by Boshen** [comment]
Related: https://github.com/web-infra-dev/oxc/issues/732

Probably need to move it to nursery before this is fixed.

**Comment by Boshen** [comment]
Globals are in!

**Comment by Boshen** [comment]
This can be fixed with the following config:

```json
{
  "env": {
    "node": true,
    "builtin": true
  }
}
```
